lms/djangoapps/courseware/migrations/0015_add_courseware_stats_index.py

```diff
@@ -1,21 +1,6 @@
 # Generated by Django 2.2.18 on 2021-02-18 17:35
 
 from django.db import migrations, models
-from django.db.utils import OperationalError
-
-def generate_index_and_ignore_error(apps, schema_editor):
-    try:
-        migrations.AddIndex(
-            model_name='studentmodule',
-            index=models.Index(fields=['module_state_key', 'grade', 'student'], name='courseware_stats'),
-        )
-    except OperationalError:
-        # This error will be thrown if the index already exists
-        pass
-
-
-def reverse_func(apps, schema_editor):
-    migrations.DropIndex(model_name='studentmodule', name='courseware_stats')
```

Comment:
is it better to do a try/except? if the index doesn't exist then simply pass

instead of removing the entire function

Reply:
we don't need revers_func for Django AddIndex migration. It will be handled by Django itself.